src/usr/local/www/diag_dns.php

```diff
@@ -84,12 +83,20 @@
 	$counter++;
 }
 
+function resolve_host_addresses($host) {	
+	$host_esc = escapeshellarg($host);
+	$resolved = array();
+	exec("/usr/bin/drill {$host_esc} A | /usr/bin/grep {$host_esc} | grep '\tA\t\|\tCNAME\t' | /usr/bin/grep -v ';' | /usr/bin/awk '{ print $5 }'", $resolved);
+	$resolved_v6 = array();
+	exec("/usr/bin/drill {$host_esc} AAAA | /usr/bin/grep {$host_esc} | grep '\tAAAA\t\|\tCNAME\t' | /usr/bin/grep -v ';' | /usr/bin/awk '{ print $5 }'", $resolved_v6);
```

Comment:
While you are touching this code, what do you think about using dns_get_record() instead shell calls to /usr/bin/drill?

http://php.net/manual/en/function.dns-get-record.php